### PR TITLE
pov: Improve code based on HLint's suggestions.

### PR DESCRIPTION
**Pull request by rbasso**
Related to #355.

### CODE REVIEW

File: exercises/pov/src/Example.hs [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
 
 reparent :: Eq a => Zipper a -> Graph a
 reparent (Zipper g []) = g
-reparent (Zipper g (c:cs)) = Graph (value g) $ (children g) ++ [reparented]
+reparent (Zipper g (c:cs)) = Graph (value g) $ children g ++ [reparented]
     where reparented = reparent (Zipper (crumbToGraph c) cs)
 
 down :: Zipper a -> Maybe (Zipper a)
-down (Zipper (Graph v (k:kids)) crumbs) = Just (Zipper k ((Crumb v [] kids):crumbs))
+down (Zipper (Graph v (k:kids)) crumbs) = Just (Zipper k (Crumb v [] kids:crumbs))
```
Comment (petertseng) [inline exercises/pov/src/Example.hs]: Do you think the spacing now makes it look deceptive? It makes it look as if the precedence is `Crumb v [] (kids:crumbs)`, perhaps a space surrounding the `:` would be fitting? What say you?

File: exercises/pov/src/Example.hs [hunk 2]
```diff
@@ -1,40 +1,40 @@
...
 down _ = Nothing
 
 right :: Zipper a -> Maybe (Zipper a)
-right (Zipper here ((Crumb v lefts (r:rights)):cs)) = Just (Zipper r (shifted:cs))
+right (Zipper here (Crumb v lefts (r:rights):cs)) = Just (Zipper r (shifted:cs))
```
Comment (petertseng) [inline exercises/pov/src/Example.hs]: same question with the space here, I think

File: exercises/pov/src/Example.hs (hunk 1 above)
Comment (rbasso) [inline exercises/pov/src/Example.hs]: I agree!
Gonna fix as soon as possible.

ps: Sorry for taking so long to answer. I'm temporarily with a very slow Internet connection (It takes at least a minute to load a GitHub page)

File: exercises/pov/src/Example.hs (hunk 2 above)
Comment (rbasso) [inline exercises/pov/src/Example.hs]: I rewrote both lines, adding spaces and substituting parenthesis by `$`.
I still don't like how it looks, but I expect it to be a little more readable.

**Pull request merged**